certmanager/README.md

```diff
@@ -8,7 +8,7 @@ in a standalone manner, separate from an SSL/TLS connection.
 ## Compiling and Running the Example
 
 ```
-$ ./configure --enable-opensslextra
+$ ./configure
```

Comment:
It looks like the example uses CRL. Perhaps you can test with that in suggest it as an option as well?

Reply:
ok.